### PR TITLE
feat: data viewer table for screenpipe data

### LINKED ISSUE #969: [bounty] pipe tableplus like data viewer table for screenpipe data
just copy tableplus/supabase like table to display your tables data

Screenshot 2024-12-14 at 10 17 04 AM

/bounty 100 

definition of done
- can paginate through stuff, does not crash your computer 
- can see data as a table
- do not display each table, the goal is to have a way for users to see their data, and that things are being captured etc
- you can use the raw sql api if it helps or use the search api 
- plz use `screenpipe-js` as much as possible check some new nextjs pipe that use it https://github.com/mediar-ai/screenpipe/tree/main/pipes/search - screenpipe-js is supposed to run both browser and server side, just some functions are not available browser side

### PR DESCRIPTION
**Pull request by neo773**
Screenshot 2024-12-16 at 1 15 53 AM

/closes #969
/claim #969

### CODE REVIEW

**Comment by louis030195** [comment]
this is wrong 

it should be outside tauri, it's a pipe not in tauri 

also should not require rust code and missing UI table

**Comment by louis030195** [comment]
can you fix? i want to merge this ASAP, taking a 15h flight in a few hours, hopefully before

**Comment by neo773** [comment]
@louis030195 
Moved to a pipe and added `ui-monitoring` table

https://github.com/user-attachments/assets/54ad74c5-07a9-4049-abb8-1979a7028e62

File: screenpipe-app-tauri/src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -111,6 +111,17 @@
         "minWidth": 1200,
         "minHeight": 850,
         "resizable": true
+      },
```
Comment (louis030195) [inline screenpipe-app-tauri/src-tauri/tauri.conf.json:114]: can you remove old stuff

**Pull request merged**